client-v2/src/components/FeedSectionHeader.tsx

```diff
@@ -136,10 +141,15 @@ const FeedSectionHeader = ({
   </SectionHeaderWithLogo>
 );
 
-const mapStateToProps = (state: State) => ({
-  isCurrentFrontsMenuOpen: selectIsCurrentFrontsMenuOpen(state),
-  frontCount: selectEditorFrontIds(state).length
-});
+const mapStateToProps = () => {
+  const selectEditorFrontsByPriority = createSelectEditorFrontsByPriority();
+  return (state: State, props: ContainerProps) => ({
```

Comment:
We need to thread the current priority through this component to the CurrentFrontList component --  it didn't feel right for such a small component to depend upon the current route.